R/plot.R

```diff
@@ -181,6 +181,37 @@ plot.tidyvpcobj <- function(x,
   g
 }
 
+#' Expand single-value vpc groups to a finite width so that they show up with `geom_ribbon()`
+#'
+#' @param vpc The vpc object
+#' @return A data frame of the vpc$strat possibly with additional rows for
+#'   single-value groups
+#' @noRd
+expand_vpc_stats_single_value <- function(vpc, xvar, width = 0.0001) {
+  d_vpc_stats <- vpc$strat
+  if (!is.null(vpc$strat)) {
+    d_vpc_stats <-
+      dplyr::grouped_df(vpc$stats, vars = names(vpc$strat)) %>%
```

Comment:
We do not use dplyr dependency in tidyvpc, but rather use data.table instead. Can you please update with data.table syntax. Thanks.

Reply:
I don't know the data.table syntax for it.  Is it something that would be simple for you to revise, or should I learn data.table?

Reply:
No problem, I can revise.

Reply:
Hi @billdenney, sorry for the delay on this. I just sat down to translate this to data.table and noticed an issue with your implementation for non-stratified vpc e.g.,

```
  d_vpc_stats <- vpc$strat
  if (!is.null(vpc$strat)) {
...
}
return(d_vpc_stats)
```

Looks like `expand_vpc_stats_single_value()` will return `NULL` when `stratify()` is not used...